example/migrations/src_migrations/main/scala/4.scala

```diff
@@ -0,0 +1,6 @@
+import slick.jdbc.H2Profile.api._
+import com.liyaos.forklift.slick.SqlResourceMigration
+
+object M4 {
+  MyMigrations.migrations = MyMigrations.migrations :+ SqlResourceMigration(4, slick.jdbc.H2Profile, MyMigrations.class)
```

Comment:
CI reports an error here. Maybe you mean `MyMigrations.getClass`?

Reply:
Saw that and checked my test sample and it should have been getClass. About to push an update